README.md

```diff
@@ -252,6 +252,16 @@ Alternatively you can assign a secret to the node to allow it to pull from your
 
 For Docker Swarm use the `--send-registry-auth` flag or its shorthand `-a` which will look up your registry credentials in your local credentials store and then transmit them over the wire to the deploy command on the API Gateway. Make sure HTTPS/TLS is enabled before attempting this.
 
+### Use faas-cli in CI environments
```

Comment:
Several paragraphs shouldn't be needed to explain that an automatic env-var changes the file-mode. Can you have a go at writing this in < 3-5 brief sentences?

Reply:
I've shortened it now, while still adding references to the links as discussed here.
https://github.com/openfaas/faas-cli/pull/831#issuecomment-703123046